.changeset/fair-dots-explode.md

```diff
@@ -0,0 +1,5 @@
+---
+'@axelar-network/axelar-cgp-sui': minor
+---
+
+Paying for gas now requires the message ticket instead of the call information.
```

Comment:
check if sui deployment/operational scripts need to be updated

Reply:
it does not